### PR TITLE
[PDI-17667] Use of vulnerable component jetty 8.1.15, CVE-2017-9735, …

### PR DESCRIPTION
**Pull request by smmribeiro**
…CVE-2017-7657 CVE-2015-2080

[PPP-4183] Use of Vulnerable Component: Multiple Jetty Components [Listed in Description] (CVE-2017-7656 | CVE-2017-7657 | CVE-2017-7658 | CVE-2015-2080 | CVE-2017-9735 )
[PDI-11851] Improve the jetty server version of Carte Server
[PPP-4402] Use of Vulnerable Component: jetty-server & jetty-util (CVE-2019-10246 | CVE-2019-10247)

To better cope with the synchronization with the Karaf upgrade, it was decided that the Jetty upgrade was to be split into two phases: the first would basically consist on centralizing the Jetty version on the parent POM (leaving the version as it is); the second phase (to be integrated into the Karaf upgrade development) would include the upgrade itself and any other changes required.

This PR belongs to phase 1 (check [maven-parent-poms#161](https://github.com/pentaho/maven-parent-poms/pull/161) for the complete list of PRs for phase 1).

**For this specific PR:**
Centralized Jetty components: use dependency management.

### CODE REVIEW

**Comment by buildguy** [comment]
### :gear: Build finished in 21m 52s with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -Dmaven-surefire-plugin.argLine=-XX:MaxRAMPercentage=90.0 -Dmaven-failsafe-plugin.argLine=-XX:MaxRAMPercentage=90.0
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4644|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6860/1/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 22m 40s with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -Dmaven-surefire-plugin.argLine=-XX:MaxRAMPercentage=90.0 -Dmaven-failsafe-plugin.argLine=-XX:MaxRAMPercentage=90.0
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4644|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6860/2/testReport)|
|----|

:information_source:  _This is an automatic message_

File: core/pom.xml [hunk 1]
```diff
@@ -353,13 +352,6 @@
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-util</artifactId>
-      <version>${jetty-util.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>*</groupId>
```
Comment (graimundo) [inline core/pom.xml]: @smmribeiro did you double check on any impact on included artifacts?

File: integration/pom.xml [hunk 2]
```diff
@@ -77,18 +76,6 @@
       <scope>test</scope>
     </dependency>
 
-    <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>test-jetty-servlet</artifactId>
-      <version>${test-jetty-servlet.version}</version>
-      <scope>test</scope>
```
Comment (graimundo) [inline integration/pom.xml]: Here you are actually changing the scope. If the scope is test here, it should be maintained as scope.

Comment (graimundo) [inline integration/pom.xml]: Did you double check the ITs to make sure it wasn't needed?

**Comment by buildguy** [comment]
### :gear: Build finished in 23m 31s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4653|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6860/17/testReport)|
|----|

:information_source:  _This is an automatic message_

**Pull request merged**